butterfree/migrations/cassandra_migration.py

```diff
@@ -0,0 +1,68 @@
+"""Cassandra Migration entity."""
+
+import warnings
+from typing import Any, Dict, List
+
+from butterfree.configs.db import CassandraConfig
+from butterfree.migrations.migration import DatabaseMigration
+
+
+class CassandraMigration(DatabaseMigration):
+    """Cassandra class for Migrations."""
+
+    @staticmethod
+    def _get_alter_table_query(columns: List[Dict[str, Any]], table_name: str) -> str:
+        parsed_columns = []
+        for col in columns:
+            parsed_columns.append(f"{col['column_name']} {col['type']}")
+
+        parsed_columns = ", ".join(parsed_columns)  # type: ignore
+
+        return f"ALTER TABLE {table_name} " f"ADD ({parsed_columns});"
```

Comment:
For cassandra we have at least 4 types of alterations allowed:
```cql
ALTER TABLE [keyspace_name.] table_name 
[ALTER column_name TYPE cql_type]
[ADD (column_definition_list)]
[DROP column_list | COMPACT STORAGE ]
[RENAME column_name TO column_name]
[WITH table_properties];
```

why only `ADD` is being covered here?

Reply:
I've added an explanation within the ```CassandraMigration``` docstring. I'll probably replicate this same logic to ```MetastoreMigration```.